docs/learn-scams.md

```diff
@@ -0,0 +1,38 @@
+---
+id: learn-scams
+title: How to Protect Yourself from Scams
+sidebar_label: How to Protect Yourself from Scams
+---
+
+Scams have unfortunately made an impact on our community. It's important to stay alert and protect
+yourself and your non-refundable crypto assets from scammers. If you ever feel you are being
+scammed, please email [our support team](mailto:support@web3.foundation).
+
+### Types of Scams
+
+- Personal messages over Telegram, and other social medias
+- 'Send us DOT/KSM, we'll send you double'
+- Emails asking for DOT/KSM addresses, posing as a member of our team
```

Comment:
Add: 
People offering to help you stake or get rewards
People answering questions you asked publicly in a private chat

Reply:
Add:

Scammers will often make it seem like the 'deal' is only available for a limited time.  Do not be tricked by this, it is always better to confirm than to risk losing everything.